packages/aragon-cli/src/commands/dao_cmds/utils/acl.js

```diff
@@ -0,0 +1,32 @@
+module.exports = ({ web3 }) => {
+  const userHasCreatePermissionRole = async daoAddr => {
+    const dao = new web3.eth.Contract(
+      require('../../apm_cmds/abi/os/Kernel').abi,
```

Comment:
We should remember to update this when the PR about moving to use package ABIs is merged.

Reply:
Good point! There should be a merge conflict reminding us to do it

Reply:
Just rebase to include the new ABI imports